src/extension.ts

```diff
@@ -316,24 +329,30 @@ function activateEvalPackage(context: vscode.ExtensionContext) {
     const provider = new JSONProvider();
     const registration = vscode.workspace.registerTextDocumentContentProvider(outputUri.scheme, provider);
 
-    const evalPackageCommand = vscode.commands.registerCommand('opa.eval.package', onActiveWorkspaceEditor(outputUri, (editor: vscode.TextEditor) => {
+    const evalPackageCommand = vscode.commands.registerCommand('opa.eval.package', onActiveWorkspaceEditor(outputUri, (editor: vscode.TextEditor, inWorkspace: boolean) => {
 
         opa.parse('opa', opa.getDataDir(editor.document.uri), (pkg: string, _: string[]) => {
 
             let args: string[] = ['eval'];
+            let inputPath = '';
 
             args.push('--stdin');
-            args.push(...opa.getRootParams());
             args.push('--package', pkg);
             args.push('--metrics');
-            args.push(...opa.getSchemaParams());
 
-            let inputPath = getInputPath();
-            if (existsSync(inputPath)) {
-                args.push('--input', inputPath);
-            } else {
-                inputPath = '';
-            }
+            ifInWorkspace(() => {
```

Comment:
The various commands that invoke `opa eval` will have to duplicate this logic. I wonder if we should factor it out into a separate function to avoid that?

Reply:
You're referring to the overlap in arguments to the `eval` command? Yea, it might be cleaner to move that into a reusable function.